### DISCUSSION TITLE
Docs subfolder sidebar_label with Autogenerated Sidebar

### DISCUSSION THREAD

**Discussion by ezbeazy** [facebook/docusaurus#8037] [Q&A]
Setting sidebar_label in the front matter of a index.md file in a subfolder causes the sidebar dropdown item to be linked and display the contents of the index.md file. 

Is there a way to change the sidebar_label of a docs/ subfolder without editing sidebar.js? and without the drop down label being linked? 

FILE STRUCTURE

docs/
  how_to/
    do_x.md
    do_y.md

SIDEBAR

How To
  How To Do X
  How To Do Y
    
Might it even be easier to have the file structure have an opinionated default that hyphens or underscores in the file structure are replaced with spaces? Or to have a boolean front matter field for 'sidebar_link'? 

I know folders allow spaces and are case sensitive, but not ideal.

**Comment by slorber** [comment]
Please create a repro, it is hard to understand what is your problem based on text

**Reply by ezbeazy** [reply]
Here you go sir 
[https://github.com/ezbeazy/docusaurus-test](https://github.com/ezbeazy/docusaurus-test)

Created the 3 possible structures of docs subfolders
[https://github.com/ezbeazy/docusaurus-test/tree/main/docs](https://github.com/ezbeazy/docusaurus-test/tree/main/docs)

Rewording the question more simply:

Is there a way to change the sidebar_label of a doc subdir in an autogenerated sidebar without the sidebar item linking to an index.md?

**Reply by ezbeazy** [reply]
sorry just noticed those links were directing to the wrong url for some reason. The urls were correct and now linked properly.
